packages/app/src/cli/services/app-logs/dev/write-app-logs.ts

```diff
@@ -27,10 +28,16 @@ export const writeAppLogsToFile = async ({
   const fullOutputPath = joinPath(getLogsDir(), path)
 
   try {
+    const payload = JSON.parse(appLog.payload)
```

Comment:
I see we're already parsing the payload [here](https://github.com/Shopify/cli/blob/204542c1938e3ce55808fce6f16ea87eca4e1563/packages/app/src/cli/services/app-logs/dev/poll-app-logs.ts#L77)- is there a clean way for us to reuse that?